markets/perps-market/contracts/storage/PerpsMarket.sol

```diff
@@ -426,6 +426,17 @@ library PerpsMarket {
         return positionPnl + fundingPnl - self.debtCorrectionAccumulator;
     }
 
+    function requiredCreditForSize(
```

Comment:
Makes sense, maybe just a note that in the future this should only be used for delta's in position size -- as short positions should still have a positive required credit.

Or perhaps this is more of a `requiredCreditForDelta` function.